__tests__/app.test.js

```diff
@@ -119,3 +119,32 @@ describe("GET /api/articles", () => {
       });
   });
 });
+
+// 5. GET /api/articles/:article_id/comments
+
+describe("GET /api/articles/:article_id/comments", () => {
+  test("1. GET: 200 returns an array of comments for the given article_id with the correct properties", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then((response) => {
+        expect(response.body.comments.length).toBe(11);
+        response.body.comments.forEach((comment) => {
+          expect(comment).toHaveProperty("comment_id", expect.any(Number));
+          expect(comment).toHaveProperty("votes", expect.any(Number));
+          expect(comment).toHaveProperty("created_at", expect.any(String));
+          expect(comment).toHaveProperty("author", expect.any(String));
+          expect(comment).toHaveProperty("body", expect.any(String));
+          expect(comment).toHaveProperty("article_id", expect.any(Number));
```

Comment:
A good happy path test, though you can be a bit more specific about what `article_id` all these comments should have 🔍